src/cli/commands/decrypt.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type, @typescript-eslint/no-use-before-define, @typescript-eslint/no-explicit-any */
+
+import { Command, flags } from '@oclif/command'
+
+import { startSpinner, failSpinner, succeedSpinner } from '../spinner'
+import { getSopsOptions, runSopsWithOptions } from '../../utils/sops'
+import { validate } from './validate'
+
+export default class Decrypt extends Command {
+  static description = 'decrypt config files'
+
+  static strict = true
+
+  static flags = {
+    help: flags.help({
+      char: 'h',
+      description: 'show help',
+    }),
+    verbose: flags.boolean({
```

Comment:
unfortunately doesn't work for me for some reason. i should have the latest and greatest, just ran:

1. `git pull`
1. `yarn`
1. `yarn build`
1. `strong-config decrypt -v blabla` and a few other syntax variations

No luck:

<img width="974" alt="image" src="https://user-images.githubusercontent.com/117495/70459156-de580500-1ab3-11ea-89ff-51845c9f9145.png">

Reply:
forgot to push that. Also investigated the verbose flag in the form `-v/-vv/-vvv` (increasing log verbosity) but it turned out that it isn't supported by oclif at the moment (https://github.com/oclif/oclif/issues/193)